### PR TITLE
Fix button doesn't work and add voltage for button

### PR DESCRIPTION
**Pull request by 5high**

### CODE REVIEW

File: devices/ITCommander.js [hunk 1]
```diff
@@ -3,12 +3,40 @@ const exposes = require('../lib/exposes');
...
+    },
+};
+
 module.exports = [
     {
         zigbeeModel: ['ITCMDR_Contact'],
         model: 'ITCMDR_Contact',
         vendor: 'IT Commander',
-        description: 'Contact Sensor',
+        description: '[Contact Sensor](https://sumju.net)',
```
Comment (Koenkk) [inline devices/ITCommander.js:39]: Please don't add link here, add them to the docs instead

File: devices/ITCommander.js [hunk 2]
```diff
@@ -3,12 +3,40 @@ const exposes = require('../lib/exposes');
...
 const reporting = require('../lib/reporting');
 
+const bind = async (endpoint, target, clusters) => {
+    for (const cluster of clusters) {
+        await endpoint.bind(cluster, target);
+    }
+};
+
+const fzclick = {
+    diyruz_freepad_clicks: {
```
Comment (Koenkk) [inline devices/ITCommander.js:13]: Please move this converter to fromZigbee.js

File: devices/ITCommander.js [hunk 3]
```diff
@@ -22,9 +50,22 @@ module.exports = [
...
+        description: '[Button](https://sumju.net)',
+        supports: 'single, double, triple, quadruple, many, hold/release',
+        fromZigbee: [fzclick.diyruz_freepad_clicks, fz.battery],
         toZigbee: [],
-        exposes: [e.action(['single', 'double', 'triple', 'hold', 'release']), e.battery()],
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(1);
+            await bind(endpoint, coordinatorEndpoint, ['genPowerCfg']);
+            device.endpoints.forEach(async (ep) => {
+                await bind(ep, coordinatorEndpoint, ['genMultistateInput']);
```
Comment (Koenkk) [inline devices/ITCommander.js:61]: Use reporting.bind instead here and remove the bind function at the top.

File: devices/ITCommander.js (hunk 2 above)
Comment (5high) [reply devices/ITCommander.js]: thanks , moved

**Pull request merged**